setup.py

```diff
@@ -26,24 +22,21 @@
     os.system('python setup.py sdist upload')
     sys.exit()
 
-packages = [
-    'wsproto',
-]
-
 setup(
     name='wsproto',
     version=version,
     description='WebSockets state-machine based protocol implementation',
     author='Benno Rice',
     author_email='benno@jeamland.net',
-    packages=packages,
+    packages=find_packages(),
```

Comment:
Ah, is that how it detects packages? I figured it would detect anything with *.py, makes sense though. Still might be prudent to only have `['wsproto']` in the case that someone adds an `__init__.py` to `test/` to allow relative imports within `test.py`. I'm fine either way though.